examples/policy_gradient.py

```diff
@@ -0,0 +1,234 @@
+"""Example of Policy Gradient in Myia.
+
+Myia is still a work in progress, and this example may change in the future.
+"""
+from __future__ import print_function
+
+from dataclasses import dataclass
+
+import gym
+import numpy as np
+import torch
+from numpy.random import RandomState
+
+from myia import ArithmeticData, myia, value_and_grad
+from myia.api import to_device
+
+# The following import installs custom tracebacks for inference errors
```

Comment:
Travis didn't publish the CI results here which I'm looking into, but it did run: https://travis-ci.org/github/mila-iqia/myia/jobs/669909314

The error is because `isort` and `black` have an incompatibility when there are comments in the middle of imports. `black` puts a newline, `isort` puts none. I say we just remove the comment.

```suggestion
```